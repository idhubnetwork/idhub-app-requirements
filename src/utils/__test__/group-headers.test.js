import groupHeaders from '../group-headers'

test('simple', () => {
  const res = groupHeaders([
    {
      depth: 2,
      value: 'title 1'
    },
    {
      depth: 3,
      value: 'title 3'
    },
    {
      depth: 3,
      value: 'title 4'
    },
    {
      depth: 2,
      value: 'title 2'
    }
  ])

  expect(res).toMatchSnapshot()
})
