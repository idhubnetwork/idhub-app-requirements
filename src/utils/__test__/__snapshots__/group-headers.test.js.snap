// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`simple 1`] = `
Array [
  Object {
    "children": Array [
      Object {
        "depth": 3,
        "value": "title 3",
      },
      Object {
        "depth": 3,
        "value": "title 4",
      },
    ],
    "depth": 2,
    "value": "title 1",
  },
  Object {
    "depth": 2,
    "value": "title 2",
  },
]
`;
